src/type/validate.ts

```diff
@@ -184,17 +184,17 @@ function validateDirectives(context: SchemaValidationContext): void {
       // Ensure the type is an input type.
       if (!isInputType(arg.type)) {
         context.reportError(
-          `The type of @${directive.name}(${arg.name}:) must be Input Type ` +
+          `The type of ${arg} must be Input Type ` +
```

Comment:
oooh this is a much nicer pattern too. Quite clever, that the arg will be schematized when it's stringified.